### ISSUE TITLE
Document compilation in conda environment

### ISSUE THREAD

**Issue by ischoegl** [Cantera/cantera-website#181]
Cantera/cantera#1191 introduced some additional simplifications for build processes started from a `conda` environment. While conda-based builds are convenient (and already actively used for the Sundials and windows-2022 runners), this capability is currently undocumented. 

From a personal perspective, I have working `conda`-based builds on both Linux (Fedora) and Windows. I assume that this will work out-of-the-box on OSX, but do not have the ability to check. Especially on Windows, the installation process is significantly easier than what is currently documented on the website.

**Comment by ischoegl** [comment]
@bryan / @decaluwe ... do either of you have any feedback on *building* within `conda` on OSX? In particular, I'd be interested whether running 
```
$ scons build && scons install
```
works without having to add any configuration options (as long as the conda environment is properly set up), and the installation subsequently being fully usable without having to modify any paths.

**Comment by speth** [comment]
I tried this on macOS, and the above steps work well, as does the Python module.

However, `scons test` fails due to some issues with library paths if any of the library dependencies (SUNDIALS, yaml-cpp, etc) are taken from the Conda install. For example, running just `scons test-general` fails with:
```
dyld[92557]: Library not loaded: @rpath/libsundials_cvodes.6.dylib
  Referenced from: /Users/speth/src/cantera/build/test/general/general
  Reason: tried: '/usr/local/lib/libsundials_cvodes.6.dylib' (no such file), '/usr/lib/libsundials_cvodes.6.dylib' (no such file)
```
Likewise, the C++ sample programs that are installed into the conda environment can be compiled easily, but then fail to run with a similar error.

This seems to be a result of SCons not wanting to specify the `rpath` on macOS in the same way that it does on Linux, even though I think the differences should be small, but haven't been able to figure out how to fix that just yet.

**Comment by ischoegl** [comment]
Great to hear that the installation works! But too bad that the library paths aren’t resolved the same way as on Linux and Windows. It’s really surprising that the test suite breaks … certainly didn’t expect that!

**Comment by speth** [comment]
The test issue should be resolved by Cantera/cantera#1229.

**Comment by ischoegl** [comment]
Turns out that [there is documentation](https://cantera.org/compiling/installation-reqs.html#conda-requirements) after all, so it just needs to be updated.
